build/scripts/tests.sh

```diff
@@ -20,14 +35,47 @@ xunit_console_version="$(get_package_version xunitrunnerconsole)"
 dotnet_runtime_version="$(get_tool_version dotnetRuntime)"
 
 if [[ "${runtime}" == "dotnet" ]]; then
-    target_framework=netcoreapp2.1
     file_list=( "${unittest_dir}"/*/netcoreapp2.1/*.UnitTests.dll )
+    file_blacklist=(
+        # Disable the VB Semantic tests while we investigate the core dump issue
+        # https://github.com/dotnet/roslyn/issues/29660
+        "Microsoft.CodeAnalysis.VisualBasic.Semantic.UnitTests.dll"
+    )
     xunit_console="${nuget_dir}"/xunit.runner.console/"${xunit_console_version}"/tools/netcoreapp2.0/xunit.console.dll
-elif [[ "${runtime}" == "mono" ]]; then
-    file_list=(
-        "${unittest_dir}/Microsoft.CodeAnalysis.CSharp.Symbol.UnitTests/net46/Microsoft.CodeAnalysis.CSharp.Symbol.UnitTests.dll"
-        "${unittest_dir}/Microsoft.CodeAnalysis.CSharp.Syntax.UnitTests/net46/Microsoft.CodeAnalysis.CSharp.Syntax.UnitTests.dll"
-        )
+elif [[ "${runtime}" =~ ^(mono|mono-debug)$ ]]; then
+    file_list=( "${unittest_dir}"/*/net46/*.UnitTests.dll )
+    file_blacklist=(
+        'Microsoft.CodeAnalysis.CSharp.Scripting.UnitTests.dll'
+        # Missing mscoree.dll, other problems
```

Comment:
Good news. I'm almost done with this now. Hope to have a PR out with the fixes EOD. Just tracking down one last hang.